actors/builtin/market/market_test.go

```diff
@@ -1506,7 +1582,7 @@ func TestLockedFundTrackingStates(t *testing.T) {
 
 	// deal1 and deal2 will now be charged at epoch curr + market.DealUpdatesInterval, so nothing changes before that.
 	rt.SetEpoch(curr + market.DealUpdatesInterval - 1)
-	actor.cronTick(rt)
+	actor.cronTick(rt, curr+market.DealUpdatesInterval-1)
```

Comment:
I haven't gone into the tests in detail since I expect them to change when the randomization moves to deal publishing, but I'm hoping that the testing cron calls will not need crafted arguments since there's nothing nondeterministic going on.